app/gist/new/route.js

```diff
@@ -7,7 +7,7 @@ export default GistRoute.extend({
   model () {
     this.store.unloadAll('gistFile');
 
-    var model = this.store.createRecord('gist', {description: 'New Twiddle'});
+    var model = this.store.createRecord('gist', {description: 'New Twiddle '});
```

Comment:
Hm, I wasn't aware the problem happened when the title had been changed. Even if the named has been changed and then later reverted back to `New Twiddle`.  Please show steps to reproduce a once edited title getting cut off.